src/opm/io/eclipse/EclOutput.cpp

```diff
@@ -367,8 +462,32 @@ std::string EclOutput::make_real_string(float value) const
     }
 }
 
+std::string EclOutput::make_real_string_ix(float value) const
+{
+    char buffer [15];
+    std::sprintf (buffer, "%10.7E", value);
```

Comment:
Suggestion: early return on special values instead of nesting `if else`:
```C++
if (value == 0)
    return " 0.000";

if (std::isnan(value))
    return "NAN";

if (std::isinf(value)) {
   if (value > 0)
      return "INF":
  else
      return "-INF";
}

// General case
#include <fmt/format.h>
return fmt::format("{:10.7E}", value);
```

Reply:
I ended up using std::sprintf because this was more efficient than std::ostringstream (as I was starting out with). Notice that this function is used a lot when for example writing a formatted unified restart file (FUNRST). 

Have you checked the performance of fmt::format relative to std::sprintf ? 
 
If not, I would like to test this before changing the code.

Agree that the code is clearer when not using nested if else. I have also removed tmpstr variable.

updated the following function 

```c++
std::string EclOutput::make_real_string_ecl(float value) const
std::string EclOutput::make_real_string_ix(float value) const
std::string EclOutput::make_doub_string_ecl(double value) const
std::string EclOutput::make_doub_string_ix(double value) const
```

Reply:
> Have you checked the performance of fmt::format relative to std::sprintf ?

I have not checked myself. The heresay on the internat is that fmtlib is performant - some numbers here: https://github.com/fmtlib/fmt